<launch>

    <arg name="port_name" value="can0" /> 
    <!-- <arg name="cmd_topic_name" default="cmd_vel" />  修改cmd话题，键盘控车 -->
    <arg name="cmd_topic_name" default="/uwb_cmd" />  <!--修改cmd话题，uwb控车 -->
    <arg name="brake_topic_name" default="/brake" /><!--修改brake话题，实时获取刹车信息 ljx-->
    <arg name="chassis_heart_topic_name" default="/chassis" />
    <arg name="chassis_status_topic_name" default="/chassis_status" />

    <node name="chassis_main_node" pkg="chassis_communication" type="chassis_communication" output="screen">

        <param name="port_name" type="string" value="$(arg port_name)" />
        <param name="cmd_topic_name" type="string" value="$(arg cmd_topic_name)" />
        <param name="brake_topic_name" type="string" value="$(arg brake_topic_name)" />
        <param name="chassis_heart_topic_name" type="string" value="$(arg chassis_heart_topic_name)" />
        <param name="chassis_status_topic_name" type="string" value="$(arg chassis_status_topic_name)" />
        
    </node>

    <!-- 只有利用 模拟 UWB数据测试时才将下方node解注释，项目交付时将下行删除，按一下动一下，上下左右键 -->
    <!-- <node name="turtle_teleop_key1" pkg="turtlesim" type="turtle_teleop_key" output="screen">
    </node> -->

</launch>
